### DISCUSSION TITLE
Method of visualising progress on activities

### DISCUSSION THREAD

**Discussion by J-Ogden** [kimai/kimai#3260] [General]
At the moment, to review progress on activites, i.e. %budget, each activity needs to be clicked on, to view its page. However, there is no summary page for a given project to visualise progress of activities relative to each other, i.e. a bar graph or a pie chart, which becomes coloured in, within each budget segment as activities progress. 
This could be a useful feature, because it ensures that time can be billed onto low utilisation activities, with overlapping time. It also enables project managers to quickly identify which activities are most ahead.

**Comment by kevinpapst** [comment]
Please add a feature request in the issues section about activity detail reports.
